ObjectFeature object_feature
string point_cloud_file_name
---
ObjectFeature object_feature